server/work-queue/jobs/send-email.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var _ = require('underscore');
+var sendmail = require('sendmail')();
+
+var log = require('../../logger')('server.work-queue.jobs.send-email');
+var config = require('../../config');
+
+var contentTmpl = _.template('Welcome to Chronicle, <%= email %>!');
+
+module.exports = {
+  // o is an object with keys { email }
+  perform: function(o, cb) {
+    var toEmail = o.email;
+    log.verbose('job created with params ' + JSON.stringify(o));
+
+    switch (config.get('env')) {
```

Comment:
Let's just stick the `local` env case inline here and remove all these other cases and remove the switch statement / env check completely. We'll get there when we get there. Travis / intern don't currently exercise the new user path, so we don't have to solve the `test` env right now.